src/index.ts

```diff
@@ -1011,12 +1023,20 @@ export class ValidationService {
 
     private untrackFormInput(form: HTMLFormElement, inputUID: string) {
         let formUID = this.getElementUID(form);
-        if (!this.formInputs[formUID]) {
+        let formInputs = this.formInputs[formUID]
+        if (!formInputs) {
             return;
         }
         let indexToRemove = this.formInputs[formUID].indexOf(inputUID);
         if (indexToRemove >= 0) {
-            this.formInputs[formUID].splice(indexToRemove, 1);
+            formInputs[formUID].splice(indexToRemove, 1);
```

Comment:
It looks like this should have been changed to:

```suggestion
            formInputs.splice(indexToRemove, 1);
```

Because `formInputs` was set to `this.formInputs[formUID]` and the previous code had this set to `this.formInputs[formUID].splice(indexToRemove, 1);`

Reply:
Good catch!